plerkle_snapshot/src/bin/solana-snapshot-etl/main.rs

```diff
@@ -28,7 +28,9 @@ struct Args {
 
 #[tokio::main(flavor = "multi_thread")]
 async fn main() -> Result<(), Box<dyn std::error::Error>> {
-    dotenvy::dotenv()?;
+    if let Err(e) = dotenvy::dotenv() {
+        warn!(error = %e, "Error initializing .env, make sure all required env is available...");
```

Comment:
do you want to exit after this log maybe? Also the tracing subscriber is not initialised yet, will this message be displayed?